code/_onclick/drag_drop.dm

```diff
@@ -23,6 +23,9 @@
 /client/MouseDown(object, location, control, params)
 	if(mouse_down_icon)
 		mouse_pointer_icon = mouse_down_icon
+	if(istype(mob.loc, /obj/vehicle/tank)) //This code is related to tank + vehicle autofire code, as seen in tank.dm
+		var/obj/vehicle/tank/our_tank = mob.loc
+		our_tank.onMouseDown(object,mob,params)
```

Comment:
You should not be having tank code here in client code

Reply:
Rohesie's autofire shit should fix this anyway

Reply:
This pr will have to wait until you don't have this code here one way or another

Reply:
If you copypaste the signal from my PR you can include it in yours.
Poke me if you need help, it's pretty trivial to do.